app/Plugin/CoreApi/Model/CoreApi.php

```diff
@@ -180,6 +180,214 @@ public function cmPluginMenus() {
+          }
+        }
+      }
+
+      // Next handle related models for CoPersonRole and OrgIdentity.
+
+      $related = array(
+        'CoPersonRole' => array(
+          'Address',
+          'AdHocAttribute',
+          'TelephoneNumber'
+        ),
+        'OrgIdentity' => array(
+          'Address',
+          'AdHocAttribute',
+          'EmailAddress',
+          'Identifier',
+          'Name',
+          'TelephoneNumber'
+        )
+      );
+
+      foreach($related as $parentModel => $relatedModels) {
+        // We use the $accessedRecords version rather than $reqData because it will
+        // have newly created parent keys (eg: for a CO Person Role that was
+        // added during this operation).
+
+        if(!empty($accessedRecords[$parentModel])) {
+          foreach($accessedRecords[$parentModel] as $id => $accessed) {
+            if($parentModel == 'OrgIdentity') {
+              // Skip read-only Org Identities
+              if($this->Co->OrgIdentity->readOnly($id)) {
+                continue;
+              }
+            }
+
+            foreach($relatedModels as $model) {
+              // While there can be multiple associated models across multiple roles,
+              // $current will have them attached to each role. Thus, we can track
+              // $accessedIds on a per-role basis, as opposed to across all roles.
+
+              $accessedIds = array();
+
+              if(!empty($accessed[$model])) {
+                foreach($accessed[$model] as $m) {
+                  $accessedIds[] = $this->upsertRecord($coId,
+                                                       $co_person_id,
+                                                       // We just need the associated model, it
+                                                       // doesn't matter how we got there so we
+                                                       // use OrgIdentity since it is the superset
+                                                        $this->Co->OrgIdentity->$model,
+                                                        $m,
+                                                        null,
+                                                        Inflector::underscore($parentModel).'_id',
+                                                        $id,
+                                                        $actorApiUserId);
+                }
+              }
+            }
+          }
+        }
+      }
+
+      // Assign Identifiers
+      // Disable provisioning since we will bulk provision at the end
+      // This will return an array describing which, if any, identifiers were assigned,
+      // but we don't do anything with the result here
+      $this->Co->CoPerson->Identifier->assign('CoPerson', $co_person_id, null, false, $actorApiUserId);
+
+      // Handle plugin models
+      // - Authenticator
+      // - XXX Clusters currently not supported
+      $not_supported_plugins = array('UnixCluster');
```

Comment:
Why aren't clusters supported? And why is only the UnixCluster plugin excluded?

Reply:
I am not sure why i added this check there. I removed  it.